### PR TITLE
[SDPTA-1019] User Site Permissions 

### PR DESCRIPTION
**Pull request by klaukl**
https://digital-vic.atlassian.net/browse/SDPTA-1019
* Add function to add all sites and site sections to demo test user so that the users can actually select site sections
* Function not required to be in tide_site as its really only used in this scenario

### CODE REVIEW

**Comment by klaukl** [comment]
Hey @vincent-gao I don't need to worry about below right?
```
  Configuration objects provided by <em class="placeholder">tide_test</em> ha  
  ve unmet dependencies: <em class="placeholder">node.type.test (wysiwyg_temp  
  late)</em>                                                                   
                                             
```

I think it only occurs when testing / reference branch. The wysiwyg_template isn't in the master branch

**Review by vincent-gao — CHANGES_REQUESTED** [review]

File: tide_demo_content.install [hunk 1]
```diff
@@ -112,3 +114,25 @@ function tide_demo_content_uninstall() {
   $repository->removeTrackedEntities();
   \Drupal::messenger()->addMessage('Removed all demo content.');
 }
+
+/**
+ * Get all sites and site sections.
+ */
+function getAllSitesAndSiteSections() {
```
[the remark this answers is not in the document]
Comment (klaukl) [reply tide_demo_content.install]: Fixed

**Comment by vincent-gao** [comment]
hey @klaukl could you please also update` tide_test` by removing `wysiwyg_template` from `node.type.test` yaml file.

**Review by vincent-gao — APPROVED** [review]

**Review by anthony-malkoun — COMMENTED** [review]
This gels with what I was seeing in profiling that it was just doing too many SQL queries.

**Pull request merged**